README.md

```diff
@@ -1,145 +1,55 @@
-# NSC CSC Website
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
 
-<div id="top"></div>
+## Computer Science Club Website
+#### North Seattle College
 
-<!-- PROJECT LOGO -->
-<br />
-<div align="center">
-<h3 align="center">NSC CSC Website</h3>
+This is the website for the Computer Science Club at North Seattle. Besides the website this project exists as a place where people new to Github, Computer Science, and building applications can get involved. 
 
-  <p align="center">
-    <description of project?>
-    <br />
-    <a href="GITHUB LINK"><strong>Explore the docs »</strong></a>
-    <br />
-    <br />
-    <a href="GITHUB LINK">View Demo</a>
-    ·
-    <a href="GITHUB LINK/issues">Report Bug</a>
-    ·
-    <a href="GITHUB LINK/issues">Request Feature</a>
-  </p>
-</div>
+### Contributing
 
-<!-- TABLE OF CONTENTS -->
-<details>
-  <summary>Table of Contents</summary>
-  <ol>
-    <li>
-      <a href="#about-the-project">About The Project</a>
-      <ul>
-        <li><a href="#built-with">Built With</a></li>
-      </ul>
-    </li>
-    <li>
-      <a href="#getting-started">Getting Started</a>
-      <ul>
-        <li><a href="#prerequisites">Prerequisites</a></li>
-        <li><a href="#installation">Installation</a></li>
-      </ul>
-    </li>
-    <li><a href="#usage">Usage</a></li>
-    <li><a href="#roadmap">Roadmap</a></li>
-    <li><a href="#contributing">Contributing</a></li>
-    <li><a href="#license">License</a></li>
-    <li><a href="#contact">Contact</a></li>
-    <li><a href="#acknowledgments">Acknowledgments</a></li>
-  </ol>
-</details>
+Do not be intimidated or dissuaded if you have little experience with any of the tools or languages involved in this project. There are multiple ways to contribute including editing documentation, design, feature requests, logging bugs, etc. 
 
-<!-- ABOUT THE PROJECT -->
+If you are a student at North Seattle College in any field we encourage you contribute! Please see `CONTRIBUTING.md` for a full outline of how to get involved.
 
-## About The Project
-
-Description of project
-
-<p align="right">(<a href="#top">back to top</a>)</p>
+### Installation
 
-### Built With
+If you are going to contribute please see `CONTRIBUTING.md` for guidelines before you go through this.
 
--   [React](https://reactjs.org/)
--   [Bootstrap](https://getbootstrap.com)
--   [NodeJS](https://nodejs.org/en)
+#### Prerequisites
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+##### MacOS
+- xcode develpment tools
+##### Windows
+- Windows Linux Subsystem
 
-### Prerequisites
-
-### Installation
+##### Then
 
 1. Contact christopher.hancock@seattlecolleges.edu for project secret keys
```

Comment:
Do we have project secrets?

Reply:
No, but we did when we put the project together. We can delete this now.